docs/modules/Observable.ts.md

```diff
@@ -13,28 +13,28 @@ Added in v0.6.0
 <h2 class="text-delta">Table of contents</h2>
 
 - [URI (type alias)](#uri-type-alias)
-- [URI](#uri)
-- [alt](#alt)
-- [ap](#ap)
-- [apFirst](#apfirst)
-- [apSecond](#apsecond)
-- [chain](#chain)
-- [chainFirst](#chainfirst)
-- [compact](#compact)
-- [filter](#filter)
-- [filterMap](#filtermap)
-- [flatten](#flatten)
-- [fromIO](#fromio)
-- [fromOption](#fromoption)
-- [fromTask](#fromtask)
-- [getMonoid](#getmonoid)
-- [map](#map)
-- [observable](#observable)
-- [of](#of)
-- [partition](#partition)
-- [partitionMap](#partitionmap)
-- [separate](#separate)
-- [toTask](#totask)
```

Comment:
oh, looks like you have a old version of `docs-ts` installed, could you please install `docs-ts@latest` and generate the docs again?

Reply:
current version in `package.json` is `0.3.4`